tasks-by-users/Rehan-Khan-Engineer/08-Custom select box/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>RK-Dropdown</title>
+    <link rel="stylesheet" href="style.css" />
+
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.6.0/font/bootstrap-icons.css"
+      integrity="sha384-7ynz3n3tAGNUYFZD3cWe5PDcE36xj85vyFkawcF6tIwxvIecqKvfwLiaFdizhPpN"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <div class="box">
+      <select name="" id="">
```

Comment:
If there is no id needed , remove id attribute @Rehan-Khan-Engineer

Reply:
Removed @Abbas4tech

Reply:
I can't see any commits regarding your changes ???
Where didi you commit those changes ??
@Rehan-Khan-Engineer